File: samples/darksome.swift
class Person {
    let name: String
    let age: Int

    init(name: String, age: Int) {
        self.name = name
        self.age = age
    }

    func sayHello() {
        print("Hello, my name is \(name) and I am \(age) years old.")
    }
}

let person1 = Person(name: "Alice", age: 25)
let person2 = Person(name: "Bob", age: 30)

print(person1.name)    // Output: "Alice"
print(person2.age)     // Output: 30

let myNumber = 42
let myArray = ["apple", "banana", "orange"]
let myString = "Hello, world!"

print(myNumber * 2)    // Output: 84
print(myArray[1])      // Output: "banana"
print(myString)        // Output: "Hello, world!"
print(myString.uppercased())  // Output: "HELLO, WORLD!"

person1.sayHello()     // Output: "Hello, my name is Alice and I am 25 years old."
person2.sayHello()     // Output: "Hello, my name is Bob and I am 30 years old."

File: samples/darksome.py
# Define a class called Person
class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    def say_hello(self):
        print(f"Hello, my name is {self.name} and I am {self.age} years old.")


# Create two objects of the Person class
person1 = Person("Alice", 25)
person2 = Person("Bob", 30)

# Access the properties of the objects
print(person1.name)  # Output: "Alice"
print(person2.age)  # Output: 30

# Declare and initialize variables with different data types
my_number = 42
my_array = ["apple", "banana", "orange"]
my_string = "Hello, world!"

# Use the variables in different ways
print(my_number * 2)  # Output: 84
print(my_array[1])  # Output: "banana"
print(my_string.upper())  # Output: "HELLO, WORLD!"

# Call the say_hello() method of the Person objects
person1.say_hello()  # Output: "Hello, my name is Alice and I am 25 years old."
person2.say_hello()  # Output: "Hello, my name is Bob and I am 30 years old."
